lib/modules/platform/gitlab/index.md

```diff
@@ -1,5 +1,11 @@
 # GitLab
 
+## Authentication
+
+First, [create a personal access token](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html) for the bot account (select `read_user`, `api` and `write_repository` scopes, or `read_user`, `read_api` and `read_repository` for dry runs).
+Configure it either as `token` in your `config.js` file, or in environment variable `RENOVATE_TOKEN`, or via CLI `--token=`.
```

Comment:
I've drafted a improved text in `lib/modules/platform/azure/index.md`, can you review this? After the review I'll make the other texts similar. 😉 

Todos:

- [ ] Get review on draft for `azure/index.md`
- [ ] Implement review comments, so that `azure/index.md` is done
- [ ] Fix Gitea
- [ ] Fix Bitbucket
- [ ] Fix Bitbucket Server
- [ ] Fix GitLab
- [ ] Fix GitHub